pkgs/applications/misc/tuba/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, vala
+, meson
+, ninja
+, python3
+, pkg-config
+, wrapGAppsHook
+, desktop-file-utils
+, gtk4
+, libadwaita
+, json-glib
+, glib
+, glib-networking
+, libxml2
+, libgee
+, libsoup
+, libsecret
+, gst_all_1
+, nix-update-script
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tuba";
+  version = "0.1.0";
+  src = fetchFromGitHub {
+    owner = "GeopJr";
+    repo = "Tuba";
+    rev = "v${version}";
+    hash = "sha256-dkURVzbDBrE4bBUvf2fPqvgLKE07tn7jl3OudZpEWUo=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    vala
+    python3
+    wrapGAppsHook
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    glib
+    glib-networking
+    json-glib
+    libxml2
+    libgee
+    libsoup
+    gtk4
+    libadwaita
+    libsecret
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-libav
+    gst-plugins-base
+    (gst-plugins-good.override { gtkSupport = true; })
```

Comment:
Isn't this the default?

Reply:
Nope: https://github.com/NixOS/nixpkgs/blob/ca4a734f86c0c64d11d8910c910c31433219c878/pkgs/development/libraries/gstreamer/good/default.nix#L32